corpus/expressions.txt

```diff
@@ -478,7 +478,8 @@ Functions
   function(arg1, ...arg2) {
     arg2;
   },
-  function stuff() {}
+  function stuff() {},
+  function trailing(a,) {}
```

Comment:
Is it worth testing more than 1 parameter here too?

Reply:
Yeah, sounds good :+1: